exchanges/execute/src/execute.ts

```diff
@@ -0,0 +1,60 @@
+import { pipe, share, filter, map, fromPromise, mergeMap, merge } from 'wonka';
+import {
+  GraphQLSchema,
+  GraphQLFieldResolver,
+  GraphQLTypeResolver,
+  execute,
+} from 'graphql';
+import { Exchange, makeResult, makeErrorResult } from '@urql/core';
+
+interface ExecuteExchangeArgs {
+  schema: GraphQLSchema;
+  rootValue?: any;
+  contextValue?: any;
+  fieldResolver?: GraphQLFieldResolver<any, any>;
+  typeResolver?: GraphQLTypeResolver<any, any>;
+}
+/** Exchange for executing queries locally on a schema using graphql-js. */
+export const executeExchange = ({
+  schema,
+  rootValue,
+  contextValue,
+  fieldResolver,
+  typeResolver,
+}: ExecuteExchangeArgs): Exchange => ({ forward }) => {
+  return ops$ => {
+    const sharedOps$ = share(ops$);
+    const targetOperationTypes = ['query', 'mutation', 'subscription'];
+
+    const executedOps$ = pipe(
+      sharedOps$,
+      filter(f => targetOperationTypes.includes(f.operationName)),
+      map(async o => {
+        try {
+          const r = await execute(
+            schema,
+            o.query,
+            rootValue,
+            contextValue,
+            o.variables,
+            null,
+            fieldResolver,
```

Comment:
These two resolvers fields shouldn’t be a thing for  `execute` and executable schemas. Can they go?

Reply:
This just proxies arguments to `graphql.js` [(source)](https://github.com/graphql/graphql-js/blob/master/src/graphql.js#L71)